src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
Per what I read from [CMake documentation](http://www.cmake.org/Wiki/CMake_Useful_Variables):
```
UNIX 
    is TRUE on all UNIX-like OS's, including Apple OS X and CygWin
WIN32 
    is TRUE on Windows. Prior to 2.8.4 this included CygWin
```
That means that I can indeed remove `NOT WIN32` check, but the `NOT APPLE` check should still be there because `APPLE` is `UNIX` to CMake, while OS X linker does not support `$ORIGIN`.